### ISSUE TITLE
WebAssembly version?

### ISSUE THREAD

**Issue by rauschma** [packem/packem#1]
Have you considered creating a WebAssembly version of Packem? I’d be curious how fast it is compared to the native versions.

**Comment by undefinedbuddy** [comment]
Yes, we've been planning to have a WASM-based version for Packem's standalone. Currently, the Windows builds are in place so perhaps we might have WASM builds after that. By the way, have you experienced any issues with the MacOS version?

**Comment by rauschma** [comment]
I haven’t used it, yet. I was mainly intrigued by the possibility of a WebAssembly version (and how portable such a version would be), given that Packem is implemented in Rust.

**Comment by undefinedbuddy** [comment]
That is entirely possible. We'll implement it soon. Thanks for your feedback.

**Comment by talentlessguy** [comment]
Any progress on this?

**Comment by Gowtham2003** [comment]
Any Progress On Webassembly Version ? 

I'm Finding Libraries Which can Bundle JS code within Browser
